src/main/java/seedu/address/logic/commands/editcommands/EditStudentCommand.java

```diff
@@ -67,6 +67,7 @@ public class EditStudentCommand extends Command {
     public static final String MESSAGE_INVALID_FIELD_STUDENT_NUMBER = "Student number should not be changed.";
     public static final String MESSAGE_DUPLICATE_FIELD_WARNING =
         "ATTENTION: No changes detected in the following fields:";
+    public static final String MESSAGE_EMAIL_DUPLICATION = "This email already exists in the address book.";
```

Comment:
Since there's already a String called MESSAGE_DUPLICATE_EMAIL, this String can be called MESSAGE_EMAIL_EXISTS instead to avoid confusion between the 2